src/Adapter/PlatformServices.Shared/netstandard1.0/Services/ns10TestContextImplementation.cs

```diff
@@ -133,6 +140,16 @@ public UTF.TestContext Context
             }
         }
 
+        public override void AddResultFile(string fileName)
+        {
+            if (string.IsNullOrEmpty(fileName))
```

Comment:
You can skip adding any code here and keep this function as a no-op for PlaformServices.Portable.
Note - PlatformServices.Portable is used just at compile time, and it will get replaced by PlatformServices.Desktop/Netcore at runtime depending on the target framework for your test project.

Since, desktop TestContext already has a implementation for AddResultsFile(), we just need to add implementation in NetCoreTestContext and we should be good to know.

Reply:
fantastic! good to know that. I'm pushing in a few mins